README.md

```diff
@@ -1,31 +1,38 @@
-#Project Name
-Hello Microverse Project
+#Project Name Hello Microverse!
+This project is the first Microverse project.  The project is done as per the requirements. I created the development branch where i introduced several changes. The changes introduced to this branch include addition of the html and css files, addition linter configuration files, rectifying errors as identified by linter, addition of descriptive README files.
 
-#Built With
+Built With
+This project was build using Html and Css. I relied on git, github, and Vs Code Studio to complete this project.
 
-    Major languages: Html, css & Javascript
-    Technologies used: VS Code, Git & Github
 
+#Getting Started
+To get a local copy up and running follow these simple example steps.
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+Authors
 
-👤 Author1
+👤@kidd254
 
-    GitHub: https://github.com/Kidd254
-    Twitter:https://twitter.com/lawrenc98789206
-    LinkedIn: https://www.linkedin.com/in/lawrence-kioko-972035240/
+    GitHub: @Kidd254
+    Twitter: @lawrence kioko
+    LinkedIn: Lawrence Kioko
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you add the social handler in the right way. 
`hint: Github: [Kidd254]( https://github.com/Kidd254)`

Reply:
@christianbiring1 thank you so much for the feedback